include/ddc/kernels/splines/spline_builder_batched.hpp

```diff
@@ -20,6 +20,8 @@ class SplineBuilderBatched
 
     using bsplines_type = typename SplineBuilder::bsplines_type;
 
+    using deriv_type = ddc::UniformPointSampling<ddc::Deriv<tag_type>>;
```

Comment:
I would not use `ddc::UniformPointSampling` this way, its attributes do not really make sense for derivatives. Actually I am wondering if the derivative dimension need attributes ? One attribute that could make sense is the order of the derivative, but is it worth it ?

Reply:
Yes I use order of derivative as attribute. I think I cannot really avoid that if I want to pass derivatives as Chunkspans in `builder/operator()` right ? ie. I may have to pass derivative of 1st order and 2nd order in X direction for all Y and Z values, in the same Chunkspan, so I need a domain associated to it.

Reply:
I speak about `derivs_domain_type` which is build from `deriv_type` and used as fondation DiscreteDomain of the chunkspans here:

`void operator()(
            ddc::ChunkSpan<double, spline_domain_type, Layout, memory_space> spline,
            ddc::ChunkSpan<double, vals_domain_type, Layout, memory_space> vals,
            std::optional<ddc::ChunkSpan<double, derivs_domain_type, Layout, memory_space>> const
                    derivs_xmin
            = std::nullopt,
            std::optional<ddc::ChunkSpan<double, derivs_domain_type, Layout, memory_space>> const
                    derivs_xmax
            = std::nullopt) const;`